action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
   sudo:
     description: 'If apt should be executed with sudo or without'
     required: false
-    default: 'true'
+    default: 'detect'
```

Comment:
I seem to have seen `auto` being more common than `detect`. @iTrooz what do you think? Should we use `auto` instead of `detect` here, too?

Reply:
Sure ! I didn't think about it, but it's a great idea